### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vbevziuk**
- [DEMO LINK](https://vbevziuk.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://vbevziuk.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.search-form__icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 50%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  transform: translate(0, -50%);
```
Comment (anastasiia-tilikina) [inline src/style.css:29]:
```suggestion
  transform: translateY(-50%);
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,35 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <label class="search-form__icon"></label>
```
Comment (anastasiia-tilikina) [inline src/index.html:20]: add `for` attribute to make label clickable

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
```
Comment (anastasiia-tilikina) [inline src/style.css:11]: Use the same units of measurement for `font-weight`, be consictent

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.search-form__search {
+  display: block;
+  padding-left: 62px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
```
Comment (anastasiia-tilikina) [inline src/style.css:45]: you don't need to hardcode height. use paddings to adjust height instead.

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  font-family: Avenir, sans-serif;
```
Comment (anastasiia-tilikina) [inline src/style.css:46]: assign default text styles on entire document and just override them where you need.

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
GJ)

let's add some fixes. check my comments.
also, placeholders' styles don't change on focus. 

image

These styles for focus in Figma show how input text should look, not placeholder.

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**